mmcv/runner/fp16_utils.py

```diff
@@ -35,7 +35,9 @@ def cast_tensor_type(inputs, src_type, dst_type):
     if isinstance(inputs, nn.Module):
         return inputs
     elif isinstance(inputs, torch.Tensor):
-        return inputs.to(dst_type)
+        # we need to confirm the type of inputs to be casted is the same as the
```

Comment:
Also update the docstring of the function to describe its behavior in detail.

Reply:
done